modules/react/card/stories/card.stories.mdx

```diff
@@ -23,46 +25,58 @@ yarn add @workday/canvas-kit-react
 
 ## Usage
 
-### Simple Card
+### Basic Example
 
-Cards have a `Card.Body` and an optional `Card.Heading`:
+Card includes a container `Card` component and the following subcomponents: `Card.Body` and
+`Card.Heading`.
 
-<ExampleCodeBlock code={Default} />
+<ExampleCodeBlock code={Basic} />
 
-### Padding
+### Depth
 
-Padding of the card can be changed via the `padding` prop:
+`Card` extends from [Box](/docs/labs-common-box--as#style-props), so it supports all style props
+that `Box` supports. This includes `depth`, which changes the Card's perceived "height" above the
+page surface by adjusting its box shadow. `Card` has a default depth of `2`.
 
-<ExampleCodeBlock code={WithNoPadding} />
+Here's an example of a `Card` with a reduced `depth` of `1`.
 
-### Depth
+<ExampleCodeBlock code={Depth} />
+
+### Padding
 
-Card also takes `depth` levels, which changes the perceived "height above the canvas" of the card.
-This changes the box shadow surrounding the card. Values are `inset`, or 0-4. Here's an inset card:
+Set the `padding` prop of the `Card` (again, supported via `Box`) to adjust its padding. Card has a
+default padding of `l`.
 
-<ExampleCodeBlock code={Depth} />
+Here's an example of a `Card` with a reduced padding of `xs`.
+
+<ExampleCodeBlock code={Padding} />
 
 ## Components
 
 ### Card
 
-The `Card` component is the container component that holds a `Card.Body` and an optional
-`Card.Heading`. The `Card` component wraps a non-semantic `div` element. The element can be replaced
-using the `as` prop or a `role` or other `aria-*` attributes can be added to give the card container
-semantic meaning. `Card` extends from [Box](/?path=/docs/labs-common-box--as#style-props), so any
-style props that `Box` supports, `Card` also supports.
+#### Usage
+
+`Card` is a container component that holds a `Card.Body` and an optional `Card.Heading`. `Card`
+wraps a non-semantic `div` element. The element can be replaced using the `as` prop, or a `role` or
```

Comment:
This is not a part of this PR, but we definitely need to find a place to explain `as` that lives outside of the Box docs, so we can reference it in places such as this.